### PR TITLE
Add model upload feature

### PR DESCRIPTION
**Pull request by naili-xing**
add mode upload feature

### CODE REVIEW

File: rafiki/custom_models_base/darknet/cfg/food/food.data [hunk 1]
```diff
@@ -0,0 +1,4 @@
+classes= 1
```
Comment (nudles) [inline rafiki/custom_models_base/darknet/cfg/food/food.data:1]: remove the foodlg.
it is confidential. so we can upload the model into the deployed version, but we cannot upload the model and data into the repo which is public.
in addition, foodlg models are just example models. why put them under custom_models_base?

File: examples/models/image_object_detection/food_darknet_xception.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
+import base64
```
Comment (nudles) [inline examples/models/image_object_detection/food_darknet_xception.py:1]: add license boilerplate to every new file you created.

File: rafiki/container/container_manager.py [hunk 3]
```diff
@@ -20,15 +20,18 @@
 import abc
 from typing import List, Dict, Tuple, Any
```
Comment (nudles) [inline rafiki/container/container_manager.py:22]: can we select the container platform (docker swarm or k8s) when we deploy rafiki?
both can work?

File: rafiki/container/docker_swarm.py [hunk 4]
```diff
@@ -226,9 +227,8 @@ def retried_func(*args, **kwargs):
                 if no == RETRY_TIMES:
                     logger.info(f'Giving up on `{func} call...')
                     raise e
-                
+
```
Comment (nudles) [inline rafiki/container/docker_swarm.py:230]: format the code to make the style consistent, tab=2 or 4? 
if you vscode, you can follow the instructions here https://github.com/apache/singa-doc/blob/master/docs-site/docs/contribute-code.md

File: scripts/kubernetes/start_redis.sh [hunk 5]
```diff
@@ -24,9 +24,9 @@ title "Starting Rafiki's Redis..."
 LOG_FILE_PATH=$PWD/logs/start_redis_service.log
 (kubectl create -f scripts/kubernetes/start_redis_service.json \
 &> $LOG_FILE_PATH) &
-ensure_stable "Rafiki's Redis Service" $LOG_FILE_PATH 10
+ensure_stable "Rafiki's Redis Service" $LOG_FILE_PATH 5
```
Comment (nudles) [inline scripts/kubernetes/start_redis.sh:27]: what does it mean by changing the number from 10 to 5?

Comment (naili-xing) [reply scripts/kubernetes/start_redis.sh:27]: wait for 5 sec is enough, we don't need to wait for that longer,

File: rafiki/container/docker_swarm.py (hunk 4 above)
Comment (naili-xing) [reply rafiki/container/docker_swarm.py:230]: ok

File: rafiki/custom_models_base/darknet/cfg/food/food.data (hunk 1 above)
Comment (naili-xing) [reply rafiki/custom_models_base/darknet/cfg/food/food.data:1]: ok,

File: rafiki/container/container_manager.py (hunk 3 above)
Comment (naili-xing) [reply rafiki/container/container_manager.py:22]: both can work

**Pull request closed without merging**